sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -689,6 +691,22 @@ private void loadFromConfiguration(Configuration configuration) {
             .get(AZURE_IDENTITY_DISABLE_MULTI_TENANT_AUTH, false);
     }
 
+    /**
+     * Gets the timeout to apply to developer credential operations.
+     * @return The timeout value for developer credential operations.
+     */
+    public Duration getDeveloperCredentialTimeout() {
+        return developerCredentialTimeout;
+    }
+
+    /**
+     * Sets the timeout for developer credential operations.
+     * @param developerCredentialTimeout The timeout value for developer credential operations.
+     */
+    public void setDeveloperCredentialTimeout(Duration developerCredentialTimeout) {
+        this.developerCredentialTimeout = developerCredentialTimeout;
```

Comment:
what if user pass in null ?

Reply:
I think it's tricky to do that given the range of computers that exist. I also think users are unlikely to do something like that and they'll catch it pretty quickly.